scripts/partition_manager.py

```diff
@@ -401,20 +402,21 @@ def align_partition(current, reqs, move_up, dynamic_partitions):
     e = 'EMPTY_{}'.format(len([x for x in reqs.keys() if 'EMPTY' in x]))
     reqs[e] = {'address': empty_partition_address,
                'size': empty_partition_size,
+               'region': reqs[dynamic_partitions[0]]['region'],
                'placement': {'before' if move_up else 'after': [current]}}
 
     if current not in dynamic_partitions:
-        # We have stolen space from the 'app' partition. Hence, all partitions which share size with 'app' partition
+        # We have stolen space from the dynamic partition. Hence, all partitions which share size with dynamic partition
```

Comment:
> We have stolen space from the dynamic partition.

This doesn't look so nice. Could it be rephrased ?
`Adding an additional dynamic partition reduces the available size for other partitions.`
or something like that, if I understand this code correctly.

Reply:
The semantics of your proposal is not correct. Since `current` is `not` in `dynamic_partitions` we have not added an additional dynamic partition. 

I will try to re-phrase.

Reply:
ok, i thought it would be added to list of dynamic partitions later,
but with the updated text I see that's not the case.
Much better. thanks.